models/pardot/model.sql

```diff
@@ -81,3 +81,24 @@ create or replace view pardot_analysis.visitoractivity_events_meta as (
   select 18, '', 'reopened opportunity'
 
 );
+
+
+create or replace view {schema}.visitoractivity as (
+  --this table has a bunch of types that really should be event actions but are very poorly formulated.
+  --the custom logic in this view is an attempt to fix that.
+  --not all of the various type / type_name combinations have been accounted for yet; I still need to determine exactly what some of them mean.
+  select
+    -- event_stream interface
+    va.created_at       as "@timestamp",
```

Comment:
one problem I can see with the `@` syntax is that MSSQL and MySQL use `@` to prefix variable names, which I believe would cause this to break.  I don't think mysql compatibility is a high priority, but if it's even a remote possibility we should try to find a more compatible solution